dashboard/app/helpers/certificates_helper.rb

```diff
@@ -16,6 +16,14 @@ def certificate_image_url(name, course, donor)
     "/certificate_images/#{encoded}.jpg"
   end
 
+  def twitter_certificate_image_url(name, course, donor)
+    return CDO.code_org_url('/images/hour_of_code_certificate.jpg', 'https:') if course.blank?
```

Comment:
I wonder if the above code still works if the `https:` parameter is passed in? Possibly something to investigate/clean up next week

Reply:
Said another way, I'm wondering if this should just be the new `certificate_image_url`

Reply:
Without this, we get `//code.org/images/...`. I'm not sure how it is working anywhere, but that's what I'm seeing

Reply:
See the `og:image` vs `twitter:image` in this pic: 
<img width="473" alt="image" src="https://github.com/user-attachments/assets/8b7ddc49-ed82-4a23-82c2-d2d1070fb248">

Reply:
@bethanyaconnor I think you are right and we should change it, but just not right now. Especially since facebook is already working and we don't want to break it

Reply:
I think this automatically adjusts the url based on whichever environment you're on (localhost, staging, prod, etc), so hypothetically it should work correctly on production.

Reply:
But since it's working with what you have, let's leave it since this is time sensitive!